### PR TITLE
STCOR-516 SWR POC

### PR DESCRIPTION
**Pull request by zburke**
`ControlledVocab` SWR POC. Yeah, I know I need tests. The originals are
failing for reasons that seem unrelated; still investigating. See also:

* [global SWR config](https://github.com/folio-org/stripes-core/pull/1015)
* [ui-users branch with this CV](https://github.com/folio-org/ui-users/pull/1636)

Refs [STCOR-516](https://issues.folio.org/browse/STCOR-516)

### CODE REVIEW

File: lib/ControlledVocabSwr/ControlledVocab.js [hunk 1]
```diff
@@ -0,0 +1,432 @@
...
+
+import { Button, Callout, Col, ConfirmationModal, Modal, Pane, Paneset, Row, Loading } from '@folio/stripes-components';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
+import { useOkapiKy } from '@folio/stripes-core';
+
+import EditableList from '../EditableList';
+import css from './ControlledVocab.css';
+import refdataActuators from './actuators-refdata';
+
+const ControlledVocab = (props) => {
```
Comment (mkuklis) [inline lib/ControlledVocabSwr/ControlledVocab.js:16]: @zburke it's nice to see a functional approach here 👍

File: lib/ControlledVocabSwr/ControlledVocab.js [hunk 2]
```diff
@@ -0,0 +1,432 @@
...
+    sortby,
+    stripes,
+    validate,
+    visibleFields,
+  } = props;
+  const ky = useOkapiKy();
+
+  // the CV things
+  const { data: values, mutate: mutateValues } = useSwr(
+    ['stripes-smart-components', 'controlled-vocab', baseUrl],
```
Comment (mkuklis) [inline lib/ControlledVocabSwr/ControlledVocab.js:50]: @zburke is this basically to make sure we create a unique key for this query?

**Review by mkuklis — APPROVED** [review]
Awesome work @zburke! It's nice to see the `swr` being used!

**Comment by zburke** [comment]
This is interesting but woefully out of date, and badly under-tested WRT the actuators-refdata functions.

**Pull request closed without merging**